src/Servers/HttpSys/src/NativeInterop/RequestQueue.cs

```diff
@@ -34,6 +39,12 @@ internal RequestQueue(UrlGroup urlGroup, string requestQueueName, RequestQueueMo
                 Created = false;
             }
 
+            if (_mode == RequestQueueMode.Receiver)
```

Comment:
```suggestion
            else if (_mode == RequestQueueMode.Receiver)
```

Reply:
Is Receiver really a distinct mode from the others? It seems like it should be a completely different option in addition to the Create/Attach paradigm. E.g. Why can't I be delegated to when I attach to an existing queue?